rise-google-sheet.html

```diff
@@ -76,7 +76,7 @@
 </dom-module>
 
 <!-- build:version -->
-<script>var sheetVersion = "2.3.0";</script>
+<script>var sheetVersion = "2.4.1";</script>
```

Comment:
@donnapep In order to log version, it has to be injected into the source. The current solution is good in that it will do that every time `gulp build` is run. The trouble is, running `gulp build` after changing package version isn't a habit for us with components because they don't necessarily require `gulp build` since there's no `dist/` folder to generate like widgets do. I'm not sure how we get around this other than just remembering to do so.